<?xml version="1.0" encoding="UTF-8"?>

<launch>

  <node pkg="imu_gnss_localizer" name="RCalc_VelocitySF" type="RCalc_VelocitySF" />
  <node pkg="imu_gnss_localizer" name="RCalc_YawrateOffsetStop" type="RCalc_YawrateOffsetStop" />
  <node pkg="imu_gnss_localizer" name="RCalc_YawrateOffset" type="RCalc_YawrateOffset1st" args="1st"/>
  <node pkg="imu_gnss_localizer" name="RCalc_YawrateOffset" type="RCalc_YawrateOffset2nd" args="2nd"/>
  <node pkg="imu_gnss_localizer" name="RCalc_Heading" type="RCalc_Heading1st" args="1st"/>
  <node pkg="imu_gnss_localizer" name="RCalc_Heading" type="RCalc_Heading2nd" args="2nd"/>
  <node pkg="imu_gnss_localizer" name="RCalc_Heading" type="RCalc_Heading3rd" args="3rd"/>
  <node pkg="imu_gnss_localizer" name="RCalc_Distance" type="RCalc_Distance" />
  <node pkg="imu_gnss_localizer" name="RCalc_Trajectory" type="RCalc_Trajectory" />
  <node pkg="imu_gnss_localizer" name="RCalc_PositionDis" type="RCalc_PositionDis" />
  <node pkg="imu_gnss_localizer" name="RCalc_PositionDis_Int" type="RCalc_PositionDis_Int" />
  <node pkg="imu_gnss_localizer" name="Debug_tool" type="Debug_tool" />

  <param name="/imu_gnss_localizer/reverse_imu" type="bool" value="true" />

</launch>
